coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/attachment/institute/ProposalDevelopmentInstituteAttachmentRule.java

```diff
@@ -145,7 +147,16 @@ private boolean validFileNameCharacters(Narrative narrative, String errorPath) {
         
         return rulePassed;
     }
-    
+
+    protected boolean validFileType(Narrative narrative, String errorPrefix) {
+        if (!Constants.PDF_REPORT_CONTENT_TYPE.equals(narrative.getContentType())) {
+            reportWarning(errorPrefix + NARRATIVE_FILE, KeyConstants.INVALID_FILE_TYPE,
+                    narrative.getName(), Constants.PDF_REPORT_CONTENT_TYPE);
+        }
+        return true;
+    }
+
+
```

Comment:
Can we add this to a service? There is a KcAttachmentService right? Unless someone else has a better place in mind. I might change the method name to validPDFfile